packages/ssr-web/app/components/card-space/user-page/index.css

```diff
@@ -52,6 +52,7 @@
   font-size: var(--boxel-font-size-xl);
   font-weight: 600;
   margin: var(--boxel-sp-xxl) auto;
+  word-break: break-all;
```

Comment:
@jurgenwerk what do you think about `overflow-wrap` vs `word-break`? I was wondering yesterday about splitting at non-space areas and whether there was anything we could do about it. However I am not very familiar with how this affects non-Latin-alphabet languages.